netkan/tests/scheduler.py

```diff
@@ -5,17 +5,19 @@
 from netkan.common import sqs_batch_entries
 from netkan.repos import CkanMetaRepo, NetkanRepo
 from netkan.scheduler import NetkanScheduler
+from netkan.cli.common import SharedArgs
 
 
 class TestScheduler(unittest.TestCase):
     test_data = PurePath(__file__).parent.joinpath('testdata', 'NetKAN')
     ckm_root = PurePath(__file__).parent.joinpath('testdata', 'CKAN-meta')
 
     def setUp(self):
-        self.nk_repo = NetkanRepo(Repo.init(self.test_data))
-        self.ckm_repo = CkanMetaRepo(Repo.init(self.ckm_root))
-        self.scheduler = NetkanScheduler(self.nk_repo, self.ckm_repo, 'TestyMcTestFace', 'token')
-        self.messages = (nk.sqs_message(self.ckm_repo.highest_version(nk.identifier))
+        common = SharedArgs()
+        common.netkan_repo = NetkanRepo(Repo.init(self.test_data))
+        common.ckanmeta_repo = CkanMetaRepo(Repo.init(self.ckm_root))
```

Comment:
Ahh I see why you configured a setter above. I'm not a huge adding code to be able to inject test data. This is probably where I'd mock the return, but that's also non-trivial.

Reply:
Yeah, I didn't like having to do a setter for this, either. I forgot that a mock object was an option! I'll change it to that...

Reply:
The mock object seems to have kept the test happy without a setter. 🎉